check/bench_test.go

```diff
@@ -0,0 +1,143 @@
+package check
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+
+const check = `---
```

Comment:
I don’t love the name `check` for this because we have `checks` within the YAML. How about `controlsYaml` or `exampleYaml`?

Reply:
Or, you could use the existing `def`?

Reply:
done , changed to controlYaml